src/components/DiffView/styles.module.scss

```diff
@@ -44,6 +44,10 @@
   .diff-line .diff-code-selected {
     background-color: $light-gray;
   }
+
+  .diff-gutter:hover button.btn {
```

Comment:
I had to do this hacky thing in order to target the button that is inside the gutter. The button is defined in a different file. Maybe there's a better way of doing this?